src/pages/messages/TasksPage.tsx

```diff
@@ -25,7 +25,7 @@ import { getLanguageString } from '../../utils/translation-helpers';
 import { TicketList } from './components/TicketList';
 import { InstitutionUser } from '../../types/user.types';
 import { dataTestId } from '../../utils/dataTestIds';
-import { StyledPageWithSideMenu, SidePanel, SideNavHeader } from '../../components/PageWithSideMenu';
+import { StyledPageWithSideMenu, SidePanel, SideNavHeader, LinkButton } from '../../components/PageWithSideMenu';
```

Comment:
Er dette et bedre navn nå? eller er det dumt å gjenbruke denne her
```suggestion
import { StyledPageWithSideMenu, SidePanel, SideNavHeader, SelectableButton } from '../../components/PageWithSideMenu';
```

Reply:
Tror det funker bra, men må vel gjøre `startIcon` optional isåfall

Reply:
Ser det blir litt kluss med teksfargen da, så lar det være slik inntil videre